packages/pf3-component-mapper/src/form-fields/select/index.js

```diff
@@ -164,6 +170,7 @@ Select.defaultProps = {
   },
   loadingMessage: 'Loading...',
   simpleValue: true,
+  pluckSingleValue: true,
```

Comment:
This could bring some incompatibility issue, if someone uses an actual array as a value for the select.

Reply:
Yep i though of that. Check the rest of the code and tests :wink:

Reply:
I thought that if someone has options like this right now:

```jsx
[
  { value:  ['apple', 'pineapple'], label: 'sweet'},
  { value: ['chocolate','kitkat'], label: 'best'},
] 
```

Then he should have to set this prop to `false` and that will break the backwards compatibility. Am I right?

Reply:
The example you posted would not work at this point because we were using `===` to select the value from options instead if `isEqual`. So nobody could use it successfully